### PR TITLE
workflows are frames; render them like they are

### PR DESCRIPTION
**Pull request by kmcq**
Consolidates rendering logic into `<Frame />`

### CODE REVIEW

File: packages/open-truss/src/configuration/engine-v1/RenderConfig.tsx [hunk 1]
```diff
@@ -41,31 +42,28 @@ export function RenderConfig({
...
   ) as FrameWrapper
   const globalContext: GlobalContext = {
     config,
     COMPONENTS,
     FrameWrapper,
   }
+  // Workflows are just frames! Use unknown to convince TS of this fact.
+  const frame = {
+    ...config,
+    view: { component: '__FRAGMENT__' },
```
Comment (kmcq) [inline packages/open-truss/src/configuration/engine-v1/RenderConfig.tsx:59]: Not sure about this hack yet. Need something to say this is top level and it's okay to not have a component

File: packages/open-truss/src/configuration/engine-v1/Frame.tsx [hunk 2]
```diff
@@ -52,6 +52,23 @@ export function Frame(props: FrameContext): React.JSX.Element {
...
+          <Frame
+            frame={subframe}
+            configPath={subframePath}
+            globalContext={props.globalContext}
+          />
+        </FrameWrapper>
+      )
+    })
+
+    if (component === '__FRAGMENT__') {
```
Comment (kmcq) [inline packages/open-truss/src/configuration/engine-v1/Frame.tsx:68]: See below comment about how I'm not sure about this

File: packages/open-truss/src/configuration/engine-v1/RenderConfig.tsx (hunk 1 above)
Comment (jonmagic) [reply packages/open-truss/src/configuration/engine-v1/RenderConfig.tsx:59]: Hacks FTW

**Review by jonmagic — APPROVED** [review]
I'm cool with the hack. We can always roll back 😄

**Review by hktouw — COMMENTED** [review]
Love this! Great to see us consolidate logic into Frame 🤘

**Pull request merged**